src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/InvitationAggregate/InvitationTests.cs

```diff
@@ -922,6 +922,32 @@ public void CompleteIpo_ShouldCompleteIpo()
             Assert.AreEqual(_currentUserId, _dutDpIpo.CompletedBy);
         }
 
+        [TestMethod]
+        public void CompleteIpo_ShouldCompleteIpo_WhenSemicolonSeparatedEmailInFunctionalRole()
+        {
+            var emails = _dutDpIpo.GetCompleterEmails();
+            Assert.IsTrue(emails.Count()==2);
+        }
+
+        [TestMethod]
+        public void CompleteIpo_ShouldCompleteIpo_WhenSemicolonSeparatedEmailInMultipleFunctionalRoles()
+        {
+            _dutDpIpo.AddParticipant(new Participant(
+                TestPlant,
+                Organization.ConstructionCompany,
+                IpoParticipantType.FunctionalRole,
+                "FRADD1",
+                null,
+                null,
+                null,
+                "fr1additional@test.com ", // Should remove whitespace.
```

Comment:
Maybe add a space in front as well as after for testing?
You are not testing that removal of whitespace (trimming) happens in any of the tests?

Reply:
Sure.